WebRoot/public/login.jsp

```diff
@@ -232,6 +232,9 @@
                             <c:set var="client" value="mobile"/>
                         </c:if>
                         <c:if test="${empty client or client eq 'preferred'}">
+                            <c:if test="${(mobileSupported or touchSupported) and zimbraxSupported}">
+                                <jsp:forward page="/public/zimbrax.jsp"/>
+                            </c:if>
                             <c:set var="client" value="${requestScope.authResult.prefs.zimbraPrefClientType[0]}"/>
```

Comment:
shouldn't we just need to update client type to zimbrax in this case instead of doing forwarding?

Reply:
removed this code and setting the client has zimbrax